### PR TITLE
change more links from http to https

### PR DESCRIPTION
**Pull request by ayan-usgs**

### CODE REVIEW

File: wqp/portal_ui/static/js/views/nldiView.js [hunk 1]
```diff
@@ -323,9 +323,9 @@ PORTAL.VIEWS.nldiView  = function(options) {
 		'World Imagery' : L.tileLayer.provider('Esri.WorldImagery')
 	};
 	var hydroLayer = L.esri.tiledMapLayer({
-		url : "http://hydrology.esri.com/arcgis/rest/services/WorldHydroReferenceOverlay/MapServer"
+		url : "https://hydrology.esri.com/arcgis/rest/services/WorldHydroReferenceOverlay/MapServer"
```
Comment (mbucknell) [inline wqp/portal_ui/static/js/views/nldiView.js]: I'm getting a time out when I changed this to https locally. http works fine. This could be a problem

Comment (ayan-usgs) [inline wqp/portal_ui/static/js/views/nldiView.js]: hmm... yeah...

**Comment by ayan-usgs** [comment]
> Just FYI: Links don't have to change to https, only ajax calls are subject to the mixed content warnings.

I know, but figure might as well use the https URL. They'll probably get redirected from http to https anyway.

File: wqp/portal_ui/static/js/views/nldiView.js (hunk 1 above)
Comment (mbucknell) [inline wqp/portal_ui/static/js/views/nldiView.js]: We may have to explore proxying this particular layer. If we do that I believe it's a new ticket.

**Pull request merged**